.github/workflows/cloud_build_failure_reporter.yml

```diff
@@ -110,7 +112,7 @@ jobs:
                     // find runs for this commit
                     for (const check of checks.data.check_runs) {
                       console.log(`found run ${check.name} for ${commit.html_url}`);
-                      if (check.name.includes('nightly')) {
+                      if (testNameList.some(substring => check.name.includes(substring))) {
```

Comment:
This is just finding one of the tests and not reporting on both of the tests

Reply:
updated logic